### PR TITLE
Master CI Pipeline badge on README.md

### LINKED ISSUE #18: Create a CI pipeline
The CI pipeline should have at least steps to run the unit tests and another to run the pep8

### PR DESCRIPTION
**Pull request by leormarqs**
Testing Snap CI tool

Fixes #18

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,3 +39,7 @@ $ gpg-group-chat --client
 # Issue board
 
 [Waffle.io/bahackers/gpg-group-chat](https://waffle.io/bahackers/gpg-group-chat)
+
+# Continous Integration Pipeline
+
+[![Build Status](https://snap-ci.com/bahackers/gpg-group-chat/branch/master/build_image)](https://snap-ci.com/bahackers/gpg-group-chat/branch/master)
```
Comment (jeffhsta) [inline README.md]: What about put this on the top of the file, just below the "GPG Group Chat"?

**Comment by leormarqs** [comment]
Creating a CI Pipeline badge on README.md for tracking branch master. 
More info on #18

File: README.md (hunk 1 above)
Comment (leormarqs) [inline README.md]: Looks good to me :+1:

**Comment by jeffhsta** [comment]
:+1:  Looks great, to me we can merge it

**Pull request merged**